### ISSUE TITLE
cannot install on Fedora 33

### ISSUE THREAD

**Issue by fangzhou-xie** [sgaure/lfe#48]
I was trying to install `lfe` on my Fedora 33 machine, but met an error and could not proceed.

```
> install.packages("lfe")
Installing package into ‘~/R/x86_64-redhat-linux-gnu-library/4.0’
(as ‘lib’ is unspecified)
trying URL 'https://cran.rstudio.com/src/contrib/lfe_2.8-6.tar.gz'
Content type 'application/x-gzip' length 1159783 bytes (1.1 MB)
==================================================
downloaded 1.1 MB

* installing *source* package ‘lfe’ ...
** package ‘lfe’ successfully unpacked and MD5 sums checked
** using staged installation
checking for gcc... gcc -m64
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... configure: error: in `/tmp/Rtmp0Pl92y/R.INSTALL31aa40faa54e/lfe':
configure: error: cannot run C compiled programs.
If you meant to cross compile, use `--host'.
See `config.log' for more details
ERROR: configuration failed for package ‘lfe’
* removing ‘~/R/x86_64-redhat-linux-gnu-library/4.0/lfe’
Warning in install.packages :
  installation of package ‘lfe’ had non-zero exit status

The downloaded source packages are in
	‘/tmp/RtmplysuMr/downloaded_packages’
```

I was trying to add `host` parameter: `install.packages('lfe', type = "source", configure.args='--host=host')` but with no luck.

I wonder if I have any missing dependencies or there are other issues with my setup. 

Thanks.

**Comment by fangzhou-xie** [comment]
Actually, I just succeeded installing it by the following command

```
install.packages('lfe', type = "source", configure.args='--host=x86_64')
```

Since I am using AMD 64 CPU, the correct `host` argument for me is `x86_64`. After specifying this, my installation was successful.

Hope it helps other people and your mileage may vary.

**Comment by fangzhou-xie** [comment]
Well, actually, it turns out that it didn't work out that way.

Although I managed to install the package, my R would crash whenever using `felm` function. This would mean the above trick is not working (although it seems to).

The only workaround I found is to install an old version:

```
install.packages("https://cran.r-project.org/src/contrib/Archive/lfe/lfe_2.0-1570.tar.gz", repos=NULL, type="source")
```

and any version after that (from 2.1 onwards) will have some error messages in compilation.
